test('Hello world', () => {

})

test('This fail', () => {
    throw new Error('Fail')
})